app/views/layouts/embed.html.erb

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Duke Digital Repository Embed</title>
+  <%= stylesheet_link_tag    'application', media: 'all' %>
+  <%= javascript_include_tag 'application' %>
+  <%= csrf_meta_tags %>
+  <%= content_for(:head) %>
+
+  <%# Separate GA account for embeds so as to not bloat visit stats every 
+  time iframe content is served up %>
+  <% if ENV['TRACK_GOOGLE_ANALYTICS'] == 'true' %>
+    <%= render 'google_analytics' %>
```

Comment:
Good point -- yes, that'd indeed be better. 

I don't see any scenario where where TRACK_GOOGLE_ANALYTICS might be False and the ID should be True or vice versa.

Yes, let's ditch the TRACK_GOOGLE_ANALYTICS variable and add a new variable for the GA DDR Embeds property.